packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -383,13 +390,25 @@ const ALARM_COLUMNS = [
   },
 ]
 
+@addSubscriptions({
+  schedules: subscribeSchedules,
+})
 @connectStore(() => {
   const getOrphanVdiSnapshots = createGetObjectsOfType('VDI-snapshot')
     .filter([_ => !_.$snapshot_of && _.$VBDs.length === 0])
     .sort()
   const getOrphanVmSnapshots = createGetObjectsOfType('VM-snapshot')
     .filter([snapshot => !snapshot.$snapshot_of])
     .sort()
+  const getLoneBackupSnapshots = createGetObjectsOfType('VM-snapshot').filter(
+    createSelector(
+      createCollectionWrapper((_, props) => map(props.schedules, 'id')),
+      scheduleIds => _ => {
+        const scheduleId = _.other['xo:backup:schedule']
+        return scheduleId !== undefined && !includes(scheduleIds, scheduleId)
```

Comment:
`scheduleId !== undefined` is not necessary as `includes` will simply return `false` since there's no `undefined` value in `scheduleIds`.